src/edu/wright/cs/raiderplanner/controller/MainController.java

```diff
@@ -99,13 +102,68 @@ public static void setSpc(StudyPlannerController newSpc) {
 	 * importing an existing Study Planner file.
 	 */
 	public static void initialise() {
-		try {
-			ui.showStartup();
-		} catch (IOException e) {
-			UiManager.reportError("Invalid file.");
-			System.exit(1);
+		boolean noAccount = false;
+		File[] files = MainController.ui.getSavesFolder().listFiles();
+		if (files != null) {
+			if (files.length == 0) {
+				noAccount = true;
+			}
+			if (files.length == 1 && files[0].getName().contains("SamplePlanner.dat")) {
+				noAccount = true;
+			}
 		}
-		// If a file is present:
+		if (noAccount) {
+			try {
+				Account newAccount = MainController.ui.createAccount();
+				StudyPlannerController study = new StudyPlannerController(newAccount);
+				// Welcome notification:
+				Notification not = new Notification("Welcome!", new GregorianCalendar(),
+						"Thank you for using RaiderPlanner!");
+				study.getPlanner().addNotification(not);
+				MainController.setSpc(study);
+				plannerFile = MainController.ui.savePlannerFileDialog();
+				loadFile(plannerFile);
+			} catch (Exception e) {
```

Comment:
A generic `Exception` should not be caught like this in most cases. You should catch the specific type of exception, like `FileNotFoundException` or `IOException` instead.